<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResPanel</class>
 <widget class="QDockWidget" name="ResPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeView" name="m_resDirView">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListView" name="m_listView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="m_actionNewShader">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/res/shader.png</normaloff>:/icon/res/shader.png</iconset>
   </property>
   <property name="text">
    <string>New Shader</string>
   </property>
  </action>
  <action name="m_actionNewMaterial">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/res/material.png</normaloff>:/icon/res/material.png</iconset>
   </property>
   <property name="text">
    <string>New Material</string>
   </property>
  </action>
  <action name="m_actionNewFolder">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/res/folder.png</normaloff>:/icon/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="m_actionShowInExplorer">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/res/folder.png</normaloff>:/icon/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>Show In Explorer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../Resource/App.qrc"/>
 </resources>
 <connections/>
</ui>
